www/src/components/repository/RepositoryDescription.js

```diff
@@ -27,7 +27,6 @@ export default function RepositoryDescription() {
         flexGrow={1}
         overflowY="auto"
         paddingBottom="xlarge"
-        marginTop="medium"
```

Comment:
Why it is removed? Was it moved somewhere else?

Reply:
That's because `PageTitle` component already adds 24px of bottom padding under horizontal line and adding an additional 16px would make it 32. According to the design, I have been following it should be 24px for all repository views.

![Zrzut ekranu z 2022-08-08 13-07-12](https://user-images.githubusercontent.com/2285385/183404855-1f89367d-4c45-462e-a060-512cf9424f0d.png)

![Zrzut ekranu z 2022-08-08 13-10-23](https://user-images.githubusercontent.com/2285385/183405010-5811c35f-1899-4da9-a784-b91aa0855298.png)

Reply:
Ok, good. This was missing few changes before. Looks to be fine now. Thanks!